src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -238,7 +240,7 @@ def load_command_table(self, _):
         g.custom_command('show', 'show_vm_nic', exception_handler=empty_on_404)
         g.custom_command('list', 'list_vm_nics')
 
-    with self.command_group('vm run-command', compute_vm_run_sdk) as g:
+    with self.command_group('vm run-command', compute_vm_run_sdk, operation_group='virtual_machine_run_commands') as g:
```

Comment:
Was introduced in 2017-03-30

Reply:
Yep, I put that as the min and max_api version in the `compute_vm_run_sdk` variable.